redash/models/changes.py

```diff
@@ -1,12 +1,20 @@
 from sqlalchemy.inspection import inspect
 from sqlalchemy_utils.models import generic_repr
+from sqlalchemy.orm import object_session
+
+from enum import Enum
 
 from .base import GFKBase, db, Column
 from .types import PseudoJSON
 
 
 @generic_repr("id", "object_type", "object_id", "created_at")
 class Change(GFKBase, db.Model):
```

Comment:
We need to add index(es) for this table to make it efficient to run a query like: `give me last N changes for object type X and id Y`.

Reply:
Currently I added only index for `object_type, object_id` pair. When I'll start working on API - I'll check if we need more indexes or not.